packages/expect/src/types.ts

```diff
@@ -134,7 +134,7 @@ type PromiseMatchers = {
   resolves: Matchers<Promise<void>> & Inverse<Matchers<Promise<void>>>;
 };
 
-export interface Matchers<R extends void | Promise<void>> {
+export interface Matchers<R extends void | Promise<void>, T = unknown> {
```

Comment:
Ah.. Seems like `Expect` type should be:

```ts

export type Expect = {
  <T = unknown>(actual: T): Matchers<void, T> &
    Inverse<Matchers<void, T>> &
    PromiseMatchers<T>;
} // ...
```

And this line probably like this:

```suggestion
// eslint-disable-next-line @typescript-eslint/no-unused-vars
export interface Matchers<R extends void | Promise<void>, _T = unknown> {
```